nrel/hive/state/simulation_state/update/step_simulation_ops.py

```diff
@@ -115,16 +87,6 @@ def perform_vehicle_state_updates(
     :return: the sim after all vehicle update functions have been called
     """
 
-    def _step_vehicle(s: SimulationState, vehicle: Vehicle) -> SimulationState:
```

Comment:
per your comment

> This smells like something a bit deep and I didn't have time to investigate further

i think it's because we were returning simulation_state here instead of "s"

Reply:
ahhhhhh that would make sense, I totally missed that on the initial inspections